vm/src/stdlib/os.rs

```
@@ -156,6 +156,12 @@ pub fn errno_err(vm: &VirtualMachine) -> PyBaseExceptionRef {
     convert_io_error(vm, io::Error::last_os_error())
 }
 
+#[derive(FromArgs, Default)]
```

clippy annotation wiill suppress the error